stylelint.config.js

```diff
@@ -1,97 +1,169 @@
+    "block-no-empty": null,
+    /* Could probably dig into this further, might be useful */
+    "no-descending-specificity": null,
+    /* Not useful at the moment */
+    "no-duplicate-selectors": null,
+    "selector-class-pattern": null,
+    "declaration-empty-line-before": null,
+    "comment-empty-line-before": [
+      "always",
+      {
+        except: ["first-nested"],
+        ignore: ["after-comment", "stylelint-commands"],
+        ignoreComments: [/^passthroughs?/],
+      },
+    ],
+    "declaration-block-no-redundant-longhand-properties": null,
+    "custom-property-empty-line-before": null,
+    "value-no-vendor-prefix": [
+      true,
+      {
+        disableFix: true,
+        severity: "warning",
+      },
+    ],
+    "max-nesting-depth": [3, { severity: "warning" }],
+    "custom-property-pattern": [/^(spectrum|mod|highcontrast|system)/, {}],
+    "alpha-value-notation": "percentage",
+    "function-no-unknown": [
+      true,
+      {
+        severity: "warning",
+      },
+    ],
+    /* @todo: would like to use "modern" eventually */
+    "color-function-notation": null,
+    "import-notation": null,
+    "property-no-unknown": [
+      true,
+      {
+        checkPrefixed: true,
+      },
+    ],
+    "declaration-block-no-duplicate-custom-properties": true,
+    "declaration-property-value-no-unknown": [
+      true,
+      {
+        ignoreProperties: {
+          transform: ["/^logical/"],
+        },
+      },
+    ],
+    "value-keyword-case": [
+      "lower",
+      {
+        camelCaseSvgKeywords: true,
+      },
+    ],
+    "selector-not-notation": "complex",
+    "header/header": [
+      join(__dirname, "COPYRIGHT"),
+      {
+        nonMatchingTolerance: 0.8,
+      },
+      {
+        fix: true,
+      },
+    ],
+    "csstools/use-logical": true,
+    /** Performance */
+    // "plugin/no-low-performance-animation-properties": [true, { severity: "warning" }],
+    "plugin/selector-bem-pattern": [
+      {
+        preset: "suit",
+        presetOptions: { namespace: "spectrum" },
+        utilitySelectors: /^\.(is|u)-[A-z0-9]+$/,
+        componentName: /^[A-Z][A-z0-9]+$/,
+      },
+      {
+        severity: "warning",
+      },
+    ],
+    /** Local/custom plugins */
+    "spectrum-tools/no-missing-parenthesis": true,
+    "spectrum-tools/no-missing-var": true,
+    /** @note this enables reporting of unused variables in a file */
+    "spectrum-tools/no-unused-custom-properties": [
+      true,
+      {
+        ignore: [/^--mod-/, /^--highcontrast-/, /^--system-/],
+        disableFix: true,
+        severity: "warning",
+      },
+    ],
+    "spectrum-tools/no-unknown-custom-properties": [
+      true,
+      {
+        /** @note this is a list of custom properties that are allowed to be unknown */
+        ignore: [
+          /^--mod-/,
+          /^--highcontrast-/,
+          /^--system-/,
+          /^--spectrum-(global|alias|component)-/,
+          /^--spectrum-animation-/,
```

Comment:
When we remove the last @spectrum-css/vars package, I think that's the time to remove these entries rather than in advance. While I like the idea that we'd catch them in migrated components, it's a nice-to-have vs. throwing a false error on un-migrated assets.